tests/test_builders/test_build_html.py

```diff
@@ -416,7 +415,7 @@ def test_html_style(app: SphinxTestApp) -> None:
     },
 )
 def test_html_sidebar(app: SphinxTestApp) -> None:
-    ctx: dict[str, Any] = {}
+    ctx: _PageContextHTML = {}  # type: ignore[typeddict-item]
```

Comment:
In this test `app.builder.add_sidebars` takes as its `ctx` parameter an empty dictionary - is that a valid use of `app.builder.add_sidebars`?

If so, it seems as though the type hint of `app.builder.add_sidebars`'s `ctx` parameter should change.

If not, it seems as though this test should change to be realistic by actually having `ctx` be a `_PageContextHTML` rather than an empty dictionary.